docs/modules/reference/pages/how-to-build-with-docker.adoc

```diff
@@ -1,19 +1,21 @@
-= Build with Docker
+= Build With Docker
 
 == Simple build
 
 The source can be compiled from source with a Docker builder image.
 
-```
+[source, console]
+----
 docker build -t myopennmshelmimage .
-```
+----
 
 will checkout the source from GitHub URL of the project, compile and installs the plugin.
 The result is a runnable image which can be executed with
```

Comment:
Should this still be included? It seems like it's out of place.

Reply:
It's a split up sentence with the code interrupting the flow. We could probably structure it better. We should create a Jira to review the entire Helm documentation set once this PR is merged.

Reply:
I want to take this whole page out. It's fairly outdated and refers to docker examples for Horizon 19.